src/python/turicreate/toolkits/sound_classifier/sound_classifier.py

```diff
@@ -11,6 +11,7 @@
 from __future__ import absolute_import as _
 
 import logging as _logging
+from math import ceil as _ceil
```

Comment:
I don't see tests being un-marked as x-fail on this PR. Am I missing something?

Reply:
Yes, please adjust the tests so that after we merge, there are only `.` and `x`, not `X` or `F`

Reply:
I wanted to fix what I can before marking xfail. Added `xfail`s in the PR